tsl/test/sql/continuous_aggs_drop_chunks.sql

```diff
@@ -78,37 +79,49 @@ CREATE MATERIALIZED VIEW conditions_7
             SUM(value), COUNT(value)
         FROM conditions GROUP BY bucket WITH DATA;
 
-CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket < 70 ORDER BY bucket;
+CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket <= 70 ORDER BY bucket;
 
 SELECT * FROM see_cagg;
 
 -- This is the simplest case, when the updated bucket is inside a chunk, so it is expected
--- that the update is always reflected after the refresh on drop.
+-- that the update is refreshed.
 UPDATE conditions SET value = 4.00 WHERE time_int = 2;
 
 -- This case updates values in the bucket, which affects two different partials in
--- two different chunks, which are dropped in the same call to drop_chunks.
+-- two different chunks.
 UPDATE conditions SET value = 4.00 WHERE time_int = 9;
 UPDATE conditions SET value = 4.00 WHERE time_int = 11;
 
+SELECT _timescaledb_internal.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 20));
```

Comment:
I am surprised that this works given that `show_chunks` returns potentially multiple rows. Is it the case that `show_chunks` here happens to return just one row?

I would have expected this:

```suggestion
SELECT _timescaledb_internal.refresh_continuous_aggregate('conditions_7', chunk)
    FROM show_chunks('conditions', 20) chunk;
```

Reply:
I also found it interesting :) I am curious if PostgreSQL has a special case for it

Reply:
It is working on a set. PG query planner knows how to plan such query correctly. We have already plenty examples including docs with other single node functions.

Reply:
See in this PR, which means that 2 chunks had invalidations:
```
    SELECT _timescaledb_internal.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 40));
   refresh_continuous_aggregate 
  ------------------------------
 
 
  (2 rows)
```